students/khtruong/lesson_04/file_lab.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+"""This module runs the file lab script.
+"""
+import os
+
+
+def printpath():
+    """Return the full path for all files in the current directory."""
+    [print(os.path.abspath(item)) for item in os.listdir()]
```

Comment:
List comprehensions are meant to be used for something else, since you're not returning anything a simple for loop would be better.

this is just a style comment, what you have is correct and valid